src/styles/main.scss

```diff
@@ -1,3 +1,215 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+$blue-color: #00acdc;
+$link-color: #060b35;
+
 body {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
   margin: 0;
 }
+
+/* Header starts */
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 0 50px;
+
+  &__logo {
+    margin: 10px 0;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav__list {
```

Comment:
Put nav__list, nav__item, etc in class nav and use &__link, &__item, etc. See more details in checklist, point 10.